collections/ghs_built-up_surface.yaml

```diff
@@ -0,0 +1,85 @@
+Name: GHS Built-up Surface
+OpenEOPID: GHS_BUILT_S
+Description: |
+  GHS built-up surface grid, derived from Sentinel2 composite and Landsat, multitemporal. 
+  This spatial raster dataset depicts the distribution of built-up surfaces, expressed as number of square metres. 
+  Data are spatial-temporal interpolated or extrapolated from 1975 to 2030 in 5 years intervals. 
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: ghs_built-up_surface/README.md
+Image: ghs_built-up_surface/GHS_Built-up_Surface.png
+EDCBrowser: ""
+Resolution: pixel size 100x100m
+GeographicalCoverage: -179.9999985, -89.0913830, 179.9991201, 89.0911561
+TemporalAvailability: 1975-01-01 - 2030-01-01
+BandInformation: GHS_BUILT_S
+Contact: "[JRC](mailto:jrc-ghsl@ec.europa.eu)"
+Provider: "[JRC](https://ghsl.jrc.ec.europa.eu/)"
+ManagedBy: "[JRC](https://ghsl.jrc.ec.europa.eu/)"

```

Comment:
`Managed by` is usually whoever added this data to the datacube, not the original data provider which is given in Provider. Please change that in all the yamls. I did some structural fixes but otherwise if this is changed it should be good!